content/blog/write-cleaner-rails-code-with-fear.md

```diff
@@ -0,0 +1,246 @@
++++
+title = "Stop Fearing nil: Write Cleaner Rails Code With Fear"
+slug = "write-cleaner-rails-code-with-fear"
+date = 2024-10-28T11:35:54+05:30
+image = "/images/2024/write-cleaner-rails-code-with-fear/header.svg"
+draft = false
+authors = ["Tilak Patidar"]
+description = ""
+tags = ["Rails", "Ruby", "Clean Code", "Fear Gem"]
+categories = ["Rails", "Ruby", "Clean Code", "Fear Gem"]
```

Comment:
existing tag and category for Rails is "Ruby on Rails", so it would be better to use that, so that all Rails blogs can be accessed through that

We can remove Fear Gem as a tag and category
This feature helps group similar things together. Fear does not qualify as a tag or category
```suggestion
tags = ["Ruby on Rails", "Ruby", "Clean Code"]
categories = ["Ruby on Rails", "Ruby", "Clean Code"]
```